go/pserver/cclient/CMakeLists.txt

```diff
@@ -9,5 +9,14 @@ project(cxx_go C Go)
 include(golang)
 include(flags)
 
-go_library(client STATIC)
+go_library(paddle_cclient STATIC)
```

Comment:
Spend a lot of time working on CMake to make it work properly.@@

This is a good suggestion,  I will rename it~